tests/test_dataset.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import os
+import numpy as np
+import nibabel as nib
+from parameterized import parameterized
+from monai.data.dataset import Dataset
+from monai.transforms.composables import LoadNiftid
+
+TEST_CASE_1 = [
+    {
+        'data': [
+            {
+                'image': 'test_image1.nii.gz',
+                'label': 'test_label1.nii.gz',
+                'extra': 'test_extra1.nii.gz'
+            },
+            {
+                'image': 'test_image2.nii.gz',
+                'label': 'test_label2.nii.gz',
+                'extra': 'test_extra2.nii.gz'
+            }
+        ],
+        'transform': LoadNiftid(keys=['image', 'label', 'extra'])
+    },
+    (128, 128, 128)
+]
+
+
+class TestDataset(unittest.TestCase):
+
+    @parameterized.expand([TEST_CASE_1])
+    def test_shape(self, input_param, expected_shape):
+        test_image = nib.Nifti1Image(np.random.randint(0, 2, size=[128, 128, 128]), np.eye(4))
+        nib.save(test_image, 'test_image1.nii.gz')
```

Comment:
Create a tmp dir to save this. Safer than saving in local dir (if program halts etc).

Reply:
Updated in the latest patch.
Thanks.